sdk/communication/AzureCommunicationChat/Tests/ChatThreadClientTests.swift

```diff
@@ -53,8 +53,8 @@ class ChatThreadClientTests: XCTestCase {
 
     override func setUpWithError() throws {
         // Initialize the chatClient
-        let endpoint = settings.endpoint ?? "https://endpoint"
```

Comment:
If the logic is the same for both, why change settings to be able to be nil?

Reply:
In the base class, settings was changed to optional, so we have to make this change here.